samples/SocialSample/Startup.cs

```diff
@@ -58,6 +61,17 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
             {
                 options.ClientId = "560027070069-37ldt4kfuohhu3m495hk2j4pjp92d382.apps.googleusercontent.com";
                 options.ClientSecret = "n2Q-GEw9RQjzcRbU3qhfTj8f";
+                options.Events = new OAuthEvents()
+                {
+                    OnRemoteError = ctx =>
```

Comment:
We opted not to have a default redirect yet as it would most likely cause a hard to understand 404. It's also not clear how many of these error text messages should be leaked to the client.